### ISSUE TITLE
False negative on `>> GITHUB_OUTPUT`

### ISSUE THREAD

**Issue by TWiStErRob** [TWiStErRob/net.twisterrob.ghlint#646]
### Reproduction steps

1. File `.github/workflows/x.yml` with the following content:
   ```yaml
   on: push
   jobs:
     repro:
       runs-on: ubuntu-latest
       steps:
         - run: echo "foo=bar" >> GITHUB_OUTPUT
   ```

2. Run `ghlint` with the following arguments:
   ```shell
   ghlint --console .github/workflows/x.yml
   ```

### Expected behavior

flagged, because it's very wrong (not just style)

### Actual behavior

🦗
